webrecorder/browsertrix
# feat: Merge workflow job types

Resolves https://github.com/webrecorder/browsertrix/issues/2073



### Changes

- Removes "URL List" and "Seeded Crawl" job type distinction and adds as additional crawl scope types instead.
- Enables specifying `DOCS_URL` in `.env`

### Manual testing

1. Log in as crawler
2. Create new workflow with each crawl scope type. Verify workflow saves and crawls as expected
3. Repeat with edit and duplicate

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Workflow Editor | <img width="455" alt="Screenshot 2024-09-24 at 8 37 45 AM" src="https://github.com/user-attachments/assets/744743bb-0304-49d4-bd29-be4152870fb3"> |



### Follow-ups

- https://github.com/webrecorder/browsertrix/issues/2090
- https://github.com/webrecorder/browsertrix/issues/2091
